spyder_env_manager/spyder/widgets/main_widget.py

```diff
@@ -231,17 +257,21 @@ def setup(self):
 
         # Options menu
         options_menu = self.get_options_menu()
-        for item in [exclude_dependency_action]:
-            self.add_item_to_menu(
-                item,
-                menu=options_menu,
-                section=SpyderEnvManagerWidgetOptionsMenuSections.Highlight,
-            )
+        self.add_item_to_menu(
+            exclude_dependency_action,
+            menu=options_menu,
+            section=SpyderEnvManagerWidgetOptionsMenuSections.PackagesDisplay,
+        )
+        self.add_item_to_menu(
+            environment_as_custom_interpreter_action,
+            menu=options_menu,
+            section=SpyderEnvManagerWidgetOptionsMenuSections.InterpreterUsage,
```

Comment:
I'm not sure if you need a new section just for this action. Please post a screenshot to check how this looks.

Reply:
![interpreter_action](https://user-images.githubusercontent.com/16781833/220204132-850d5d6f-0c2a-4ab6-83f7-13977ca59f72.png)

Reply:
Ok, that's what I was imagining. The thing is having too many sections in a menu makes them loose their meaning a bit.

So, according to that and the screenshot you posted, I suggest to do the following changes:

* To me, importing and exporting are more important actions than `Exclude dependency packages` and `Set current environment`. So, I think we should change the actions' ordering to be like this

    ```
    Import ...
    Export ...
    ----------
    Set current environment ...
    Exclude dependency packages
    ----------
    ...
    ```

* I think `Set current environment` should be a checkbox, so that when it's enabled, it always set the current environment to the one selected in Spyder-env-manager. Otherwise, users would need to go to the menu and click the action to have the current environment as Spyder's Python interpreter, right?